tests/unit/components/cylc/toolbar.vue.spec.js

```diff
@@ -74,78 +63,25 @@ describe('Toolbar component', () => {
     ]
     store.state.workflows.workflowName = 'test'
   })
-  it('should initialize props', () => {
-    const wrapper = shallowMount(Toolbar, {
-      localVue,
-      vuetify,
-      store,
-      mocks: {
-        $route
-      }
-    })
-    expect(wrapper.element.children[0].className).to.equal('c-toolbar')
+  it('should mount the component', async () => {
+    const wrapper = mountFunction()
+    await wrapper.vm.$nextTick()
+    expect(wrapper.vm.$el).to.not.equal(null)
   })
   it('should hide and display drawer according to screen viewport size', async () => {
-    const wrapper = mount(Toolbar, {
-      localVue,
-      vuetify,
-      store,
-      mocks: {
-        $route,
-        $vuetify: {
-          application: {
-            register: () => {}
-          }
-        },
-        $t: () => {} // vue-i18n,
-      }
+    const wrapper = mountFunction()
+    await wrapper.setData({
+      responsive: false
     })
     expect(store.state.app.drawer).to.equal(null)
     // empty wrapper before responsive is set to true
     expect(wrapper.find('button.default').exists()).to.equal(false)
     // let's make it responsive, so that the burger menu is visible
-    wrapper.vm.$data.responsive = true
-    await wrapper.vm.$nextTick()
+    await wrapper.setData({
+      responsive: true
+    })
```

Comment:
The `setData` function does pretty much the same I was doing before, but it i) encapsulates the behaviour (so if Vue Test Utils changes it, I won't have to learn how to sync data in wrappers), and ii) takes care of refreshing the data state for the test (I was waiting for the next tick, but not sure if that's enough, or if I had to wait on more things) - https://vue-test-utils.vuejs.org/api/wrapper/#setdata